### PR TITLE
update-temp(zkevm): using zkEVM quickstart guide to capture content for ethglobal nyc hackathon

### PR DESCRIPTION
**Pull request by DannyS03**
## Context

- Utilize the zkEVM Quickstart Guide to gather content for the ETHGlobal NYC Hackathon as there was a printing issue with the flyers where the various QR codes are duplicated (and thus point to the same endpoint for different sections.)
- Timeline: until the end of the hackathon on September 24th.

### CODE REVIEW

**Comment by teeolendo** [comment]
I've reviewed the repo and the ETH Global section should be:

`
# ETHGlobal NYC Hackathon

---

# Get Started with zkEVM

Welcome! Follow the steps below to deploy dApps and win bounties!
If you need any help, head over to the Polygon booth or reach out to the team in the
ETHGlobal Polygon Discord channel.

---
## Step 1: Quickstart
---

Deploy a sample application in minutes, interact seamlessly with the zkEVM network, and launch your own zkEVM instance by following the steps outlined below:

### Link and Launch: Execute a Sample dApp

1. **[<ins>Connect your wallet to the zkEVM network</ins>](#connecting-to-zkevm)** to initiate interaction with the network.
2. **[<ins>Deploy a demo smart contract to zkEVM</ins>](#deploying-smart-contracts)** as a quick way to get started, and then consider exploring **[<ins>more smart contract examples</ins>](/docs/category/smart-contracts/)** for a deeper dive into deploying to zkEVM.

---
## Step 2: Get Testnet ETH
---

Refer to the **[<ins>Faucet Guide</ins>](/docs/zkevm/guides/zkevm-faucet/#using-the-polygon-zketh-faucet)** to acquire Testnet ETH. After obtaining some, you have the option to **[<ins>bridge some of your Goerli tokens</ins>](/docs/zkevm/bridge-to-zkevm/)** or initiate a request via XMTP.

---
## [BONUS] Step 3: Extend the LXLY Bridge Implementation
---

Develop and enhance the LXLY sample implementation by incorporating support for cross-chain messaging and asset transfers.

You can check out the [code samples](https://github.com/0xPolygonHermez/code-examples) and refer to the specifications of the bridge **[<ins>here</ins>](/docs/category/zkevm-bridge/)** for detailed information.

---
## Step 4: Explore Polygon ID
---

Finally, refer to the **[<ins>official Polygon ID developer documentation</ins>](https://devs.polygonid.com/)** to understand how to verify and issue credentials. Initially, you will need to set up a Polygon ID wallet. To accomplish this, follow the **[<ins>Quickstart Demo</ins>](https://devs.polygonid.com/docs/quick-start-demo/)** and continue the guide to issue credentials.

Check out these code samples that demonstrate Verifiable Credential Issuance and Verification:

- [Lens ID Issuer](https://github.com/teeolendo/lens-id/)
- [VC Gated Dapp](https://github.com/oceans404/fullstack-polygon-id-vc-gated-dapp)

---
---`

File: docs/zkevm/develop.md [hunk 1]
```diff
@@ -11,6 +11,57 @@ keywords:
...
+---
+
+Deploy a sample application in minutes, interact seamlessly with the zkEVM network, and launch your own zkEVM instance by following the steps outlined below:
+
+### Link and Launch: Execute a Sample dApp
+
+1. **[<ins>Connect your wallet to the zkEVM network</ins>](#connecting-to-zkevm)** to initiate interaction with the network.
+2. **[<ins>Deploy a demo smart contract to zkEVM</ins>](#deploying-smart-contracts)** as a quick way to get started, and then consider exploring **[<ins>more smart contract examples</ins>](/docs/category/smart-contracts/)** for a deeper dive into deploying to zkEVM.
+
+### Configure and Deploy: Spin up the Network
```
Comment (teeolendo) [inline docs/zkevm/develop.md:35]: Devs won't be spinning up a new node

File: docs/zkevm/develop.md [hunk 2]
```diff
@@ -11,6 +11,57 @@ keywords:
...
+
+Refer to the **[<ins>Faucet Guide</ins>](/docs/zkevm/guides/zkevm-faucet/#using-the-polygon-zketh-faucet)** to acquire Testnet ETH. After obtaining some, you have the option to **[<ins>bridge some of your Goerli tokens</ins>](/docs/zkevm/bridge-to-zkevm/)** or initiate a request via XMTP.
+
+---
+## [BONUS] Step 3: Extend the LXLY Bridge Implementation
+---
+
+Develop and enhance the LXLY sample implementation by incorporating support for various types of cross-chain messaging and asset transfers.
+
+Refer to the specifications of the bridge **[<ins>here</ins>](/docs/category/zkevm-bridge/)** for detailed information.
```
Comment (teeolendo) [inline docs/zkevm/develop.md:52]: Please add the following to show the LxLy code samples

`You can check out the [code samples](https://github.com/0xPolygonHermez/code-examples) and refer to the specifications of the bridge **[<ins>here</ins>](/docs/category/zkevm-bridge/)** for detailed information.`

File: docs/zkevm/develop.md [hunk 3]
```diff
@@ -11,6 +11,57 @@ keywords:
...
+
+---
+## Step 4: Explore Polygon ID
+---
+
+Finally, refer to the **[<ins>official Polygon ID developer documentation</ins>](https://devs.polygonid.com/)** to understand how to verify and issue credentials. Initially, you will need to set up a Polygon ID wallet. To accomplish this, follow the **[<ins>Quickstart Demo</ins>](https://devs.polygonid.com/docs/quick-start-demo/)** and continue the guide to issue credentials.
+
```
Comment (teeolendo) [inline docs/zkevm/develop.md:59]: Add this as well:

`Check out these code samples that demonstrate Verifiable Credential Issuance and Verification:
- [Lens ID Issuer](https://github.com/teeolendo/lens-id/)
- [VC Gated Dapp](https://github.com/oceans404/fullstack-polygon-id-vc-gated-dapp)
`

**Review by teeolendo — APPROVED** [review]

**Review by teeolendo — APPROVED** [review]

**Review by teeolendo — APPROVED** [review]

**Review by cerberushades — APPROVED** [review]

**Pull request merged**